Dockerfile

```diff
@@ -58,6 +58,9 @@ RUN apt-get update -qq \
     && apt-get install -y xmlstarlet \
     && rm -rf /var/lib/apt/lists/*
 
+# # Install unzip for LOGBACK_LEVEL case
+RUN apt-get update && apt-get install unzip
```

Comment:
It would be better to combine this with the existing `apt-get` line above, which also already invokes `apt-get update`.